$OpenBSD: patch-deps_uv_src_unix_openbsd_c,v 1.6 2018/09/19 19:10:27 abieber Exp $

Fix from Fabian Raetz <fabian.raetz@gmail.com>, currently set for 
the next release of 1.x: https://github.com/libuv/libuv/pull/652

Index: deps/uv/src/unix/openbsd.c
--- deps/uv/src/unix/openbsd.c.orig
+++ deps/uv/src/unix/openbsd.c
@@ -222,7 +222,7 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count)
   int numcpus = 1;
   int which[] = {CTL_HW,HW_MODEL,0};
   size_t size;
-  int i;
+  int i, j;
   uv_cpu_info_t* cpu_info;
 
   size = sizeof(model);
